lib/build.js

```diff
@@ -40,14 +40,18 @@ function getBlockedByIds(pipeline, job) {
         return Promise.resolve(blockedByIds);
     }
 
+    const externalBlockedByNames = blockedByNames.filter(
+        name => name.startsWith('~sd@'));
+
+    for (let i = 0; i < blockedByNames.length; i += 1) {
```

Comment:
`blockedByNames` still contains `~sd@` entries, however `findIdsOfMatchedJobs` filters it out.

So this block can be reduced to `blockedByNames = blockedByNames.map(name => name.replace('~', ''));`